distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -838,6 +855,28 @@ type Table
             new_type_ref = SQL_Type_Reference.from_constant sql_type
             Column.Value ("Constant_" + UUID.randomUUID.to_text) self.connection new_type_ref expr self.context
 
+    ## PRIVATE
+       Create a unique temporary column name.
+    make_temp_column_name : Text
+    make_temp_column_name = ("Temp_" + UUID.randomUUID.to_text)
```

Comment:
Please use `self.column_naming_helper.create_unique_name_strategy` to ensure that the generated column name does not clash with existing column names and keeps the length limit.

Reply:
The whole method is probably best moved into `Column_Naming_Helper`

Reply:
Done